### DISCUSSION TITLE
DPS Meter

### DISCUSSION THREAD

**Discussion by philmae** [smasherprog/EqTool#18] [Ideas]
Currently, as soon as you stop DPSsing on a mob, the meter resets.

Also, dont reset the DPS meter immediately. You barely have a chance to see the dps after the fight and click copy. Very hard to do it in time

I am sure you can store the fights for a later copy paste as well. As they are in the logs, you should be able to search back 1day and extract all fights with dps.

**Comment by smasherprog** [comment]
The time limit its 20 seconds after activity stops on a mob. I can make this a setting where you can extend this for as long as you want.
If you are on a raid having a long timeout will cause the DPS window to become overwhelmed with information. If you are in a group or solo a longer duration is fine. maybe I need to dynamically set the timeout based on how many active mobs are engaged....

I did store the fight information at one point locally in a csv file. The code is still there so I can add an option to save the fight logs to disk.

**Reply by philmae** [reply]
Yeah, to set this timeout dynamically would probably be the way to go. As well as you mention already, if you can store the fight information locally (as an optional thingy), that would even be better. 

I have been using such a "log reading" tool in EQ2 extensively for DPS meters, and what I can vaguely remember is that it categorized the fights based on mob, similar to the spell/trigger window you have. So you can just go back in the tool in history per encounter and see what the dps was on that mob during the last 2-3 hrs of raids, copy paste them or whatever. Maybe the storing the historical DPS is something for an additional window, if you think this is to heavy for the normal DPS window, although I am sure there is a smart way to do this (as I have seen it for eq2 running flawlessly)

**Comment by smasherprog** [comment]
extended timeout to 40 seconds for all for now
